Xamarin.Essentials/Geolocation/Geolocation.mac.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Geolocation
```

Comment:
`CoreLocation` is supported on Mac and can be used for this.

Reply:
Added.